extension/lib/main.js

```diff
@@ -115,7 +115,7 @@ exports.main = function (options, callbacks) {
     }
   });
 
-  memory.reporter.on(config.application.topic_memory_statistics, function (aData) {
```

Comment:
These changes aren't really necessary for the `EventEmitter` rewrite.

It just would help make `unload`'s work easier, if we would `xxxxx.reporter.off` at module unload.